ui/src/components/table/TableExpansionRow.jsx

```diff
@@ -4,14 +4,15 @@ import Table from '@splunk/react-ui/Table';
 import styled from 'styled-components';
 import { _ } from '@splunk/ui-utils/i18n';
 
+// eslint-disable-next-line import/no-cycle
```

Comment:
that sounds not right. Can you move the common function to a separate file instead?

Reply:
Done